tests/tests.js

```diff
@@ -98,87 +75,79 @@ describe('Discord Tests', function() {
     /**
      * Tests related to pull requests sent to the hook
      */
-    describe('Hook Tests', function() {
-
-        var samplePayload = getFileContents('test1', 'payload');
-        var sampleHeaders = getFileContents('test1', 'headers');
+    describe('Basic Hook Tests', function() {
 
         /**
          * Test that the hook returns a 200 when posted to
-         */
+        */
         it('Hook returns a 200 OK when posted to', function(done) {
-            sendPost(function(error, response, body) {
+            sendPost('1', function(error, response, body) {
                 assert.ok(!error && response.statusCode === 200 && body === 'OK');
                 done();
             });
         });
 
-        /**
-         * Tests that the "test1" PR performs the functions which its conditions match:
-         *      -  A pull request with 1 commit
-         *      -  The commit has one file
-         *      -  The file has one CSS property which should trigger a warning
-         *      -  If everything runs its course, there should be one call to the commenting endpoint
-         */
-        it('Running simplest Discord case: 1 commit, 1 file, 1 expected comment', function(done) {
-            var repoFullName = samplePayload.pull_request.base.repo.full_name;
-
-            var commitsPayload = getFileContents('test1', 'commits');
-            var commit1Payload = getFileContents('test1', 'commit1');
-            var commit1Contents = getFileContents('test1', 'commit1-contents');
-            var prNumber = samplePayload.pull_request.number;
-
-            setupNock(urlPatterns.commits, {
-                repo: repoFullName,
-                number: prNumber
-            }, commitsPayload);
-
-            setupNock(urlPatterns.commit, {
-                repo: repoFullName,
-                sha: commit1Payload.sha
-            }, commit1Payload);
-
-            setupNock(urlPatterns.contents, {
-                repo: repoFullName,
-                path: commit1Payload.files[0].filename,
-                branch: samplePayload.pull_request.head.ref
-            }, commit1Contents);
-
-            // Since the goal (success) is posting one comment, if this URL is matched, we've succeeded
-            setupNock(urlPatterns.comment, {
-                repo: repoFullName,
-                number: prNumber
-            }, '', done, 'post');
-
-            sendPost();
-        });
+    });
 
-        /**
-         * Sends a basic POST to the discord hook endpoint
-         */
-        function sendPost(cb) {
-            request.post({
-                url: appHost + '/hook',
-                headers: sampleHeaders,
-                form: samplePayload
-            }, cb);
-        }
+    /**
+     * Run tests on recorded PRs
+    */
+    describe('Recorded Hook Tests', function() {
```

Comment:
These automated tests encompass what their description says within the command line